docs/workflow_solutions/using_anaconda.md

```diff
@@ -260,6 +260,273 @@ Now we can be sure that the correct versions of the software will be installed o
+
+```bash
+
+conda list | grep "torch"
+
+```
+
+You should get an output like the below image.
+
+![!PyTorch Env Output](images/pytorchversion_output.png)
+
+The same process can be followed for installing another Deep Learning library Tensorflow (see instructions [below](..//workflow_solutions/using_anaconda.md#steps-to-install-tensorflow)) with some minute differences. You may decide to install the TensorFlow library into the same environment or create a new one. As a best practice, you may want to install these libraries in different environments.
+
+### Using PyTorch on Jupyter Notebook
+
+As an example we will be using a sample Jupyter Notebook with just a simple torch function to test if a GPU will be utilized with PyTorch functions. Run the command in a cell, and if your output is `True`, then you have your GPU setup to support PyTorch functions.
+
+```bash
+
+import torch
+
+print(torch.cuda.is_available())
+
+x = torch.cuda.current_device()
+
+print(torch.get_device_name(x))
+
+```
+
+![!PyTorch Jupyter Notebook Output](images/pytorch_output.png)
+
+### Steps to Install TensorFlow CPU and GPU Versions Using Cheaha CLI
+
+1. Create a new environment that is compatible with supported tensorflow versions, use the below command to do this. For this tuorial we use Python 3.11.
+
+    ```bash
+
+    conda create -n tensorflow python=3.11
+
+    ```
+
+2. The TensorFlow CPU and GPU versions requires pip to be up-to-date, to install and upgrade pip to the latest version use the below command.
+
+    ```bash
+
+    pip install -—upgrade pip
+
+    ```
+
+3. Install TensorFlow with pip
+
+    ```bash
+
+    pip install tensorflow[and-cuda]
+
+    ```
+
+The image below shows an output that the TensorFlow library will utilize the available GPU.
+
+![TensorFlow GPU](images/tensor_gpu.png)
+
+### Update packages in an environment
+
+To ensure packages and their dependencies are all up to date, it is a best practice to regularly update installed packages, and libraries in your activated environment.
+
+```bash
+
+conda update -—all
+
+```
+
+### Deleting an Environment
+
+To delete an environment, use the following command. Remember to replace `<env>` with the existing environment name.
+
+```bash
+
+conda env remove —-name <env>
+
+```
+
+### Sharing your environment file
+
+To share your environment for collaboration, there are primarily 3 ways to export environments, the below commands show how to create environment files that can be shared for replication. Remember to replace `<env>` with the existing environment name.
+
+1. Cross-Platform Compatible
+
+    ```bash
+
+    conda env export --from-history > <env>.yml 
+
+    ```
+
+1. Platform + Package Specific
+
+Create .yml file to share, replace `<envname>` (represents the name of your environment) and `<env>` (represents the name of the file you want to export) with preferred names for file.
```

Comment:
Be sure to indent consistently. Is this paragraph meant to be a part of the numbered item above it? If so it needs to be indented. I assume the fenced code block below does as well.